client/components/plans/plan/index.jsx

```diff
@@ -249,6 +250,21 @@ module.exports = React.createClass( {
 		);
 	},
 
+	getFreeDomainMessage: function() {
+		if ( abtest( 'promoteFreeDomain' ) !== 'freeDomain' ) {
+			return null;
+		}
+
+		if ( this.props.plan && isPlan( this.props.plan ) ) {
+			return (
+				<div className="plan__free-domain-message">
+					<Gridicon icon="checkmark" size="12" />
```

Comment:
Brie is right, we should use a number since that's [what the component expects](https://github.com/Automattic/wp-calypso/blob/6dde06cd386d21f5bd18488cb88a26c493b57c05/client/components/gridicon/index.jsx#L30).